swiperefresh/src/main/java/com/google/accompanist/swiperefresh/SwipeRefresh.kt

```diff
@@ -223,12 +287,21 @@ fun SwipeRefresh(
     state: SwipeRefreshState,
     onRefresh: () -> Unit,
     modifier: Modifier = Modifier,
+    position: Position = Position.TOP,
     swipeEnabled: Boolean = true,
     refreshTriggerDistance: Dp = 80.dp,
-    indicatorAlignment: Alignment = Alignment.TopCenter,
+    indicatorHorizontalAlignment: Alignment.Horizontal = Alignment.CenterHorizontally,
     indicatorPadding: PaddingValues = PaddingValues(0.dp),
     indicator: @Composable (state: SwipeRefreshState, refreshTrigger: Dp) -> Unit = { s, trigger ->
-        SwipeRefreshIndicator(s, trigger)
+        SwipeRefreshIndicator(
+            state = s,
+            refreshTriggerDistance = trigger,
+            senseOfRotation = if (position == Position.TOP) {
```

Comment:
I think this could just be `clockwise: Boolean`. The enum isn't adding anything here, but does bloat the API.